mmdet/models/necks/ct_resnet_neck.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+import torch.nn as nn
+from mmcv.cnn import ConvModule
+from mmcv.runner import BaseModule, auto_fp16
+
+from mmdet.models.builder import NECKS
+
+
+@NECKS.register_module()
+class CTResNetNeck(BaseModule):
+    """The neck used in `CenterNet <https://arxiv.org/abs/1904.07850>`_ for
+    object classification and box regression.
+
+    Args:
+         in_channel (int): Number of input channels.
+         num_deconv_filters (tuple[int]): Number of filters per stage.
+         num_deconv_kernels (tuple[int]): Number of kernels per stage.
+         use_dcn (bool): If True, use DCNv2. Default: True.
+         init_cfg (dict or list[dict], optional): Initialization config dict.
+    """
+
+    def __init__(self,
+                 in_channel,
+                 num_deconv_filters,
+                 num_deconv_kernels,
+                 use_dcn=True,
+                 init_cfg=None):
+        super(CTResNetNeck, self).__init__(init_cfg)
+        assert len(num_deconv_filters) == len(num_deconv_kernels)
+        self.use_dcn = use_dcn
+        self.in_channel = in_channel
+        self.deconv_layers = self._make_deconv_layer(num_deconv_filters,
+                                                     num_deconv_kernels)
+
+    def _make_deconv_layer(self, num_deconv_filters, num_deconv_kernels):
+        """use deconv layers to upsample backbone's output."""
+        layers = []
+        for i in range(len(num_deconv_filters)):
+            feat_channel = num_deconv_filters[i]
+            conv_module = ConvModule(
+                self.in_channel,
+                feat_channel,
+                3,
+                padding=1,
+                conv_cfg=dict(type='DCNv2') if self.use_dcn else None,
+                norm_cfg=dict(type='BN'))
+            layers.append(conv_module)
+            upsample_conv = nn.ConvTranspose2d(
```

Comment:
ConvModule support to build `ConvTranspose2d`, so we can use ConvModule to simplify the build of upsample_conv